.travis.yml

```diff
@@ -1,25 +1,26 @@
-language: rust
+language: minimal
 cache:
   yarn: true
   directories:
   - "$HOME/.cargo"
   - "$HOME/.cache/sccache"
   - "$TRAVIS_BUILD_DIR/api_tests/node_modules"
 sudo: required
-rust: nightly-2018-12-06
```

Comment:
This has the nice side-effect that we are no longer duplicating the used Rust version :)